reana_server/config.py

```diff
@@ -300,9 +301,14 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
 
 # Workspace retention rules
 # ==================
-
-WORKSPACE_RETENTION_PERIOD = int(os.getenv("WORKSPACE_RETENTION_PERIOD", "365"))
-"""Maximum allowed period for workspace retention rules."""
+_workspace_retention_period_env = os.getenv("WORKSPACE_RETENTION_PERIOD", "365")
```

Comment:
```suggestion
_workspace_retention_period_env = os.getenv("WORKSPACE_RETENTION_PERIOD", "forever")
```
WDYT? We would also need to update the value in the Helm chart's `values.yaml`.

Reply:
I will update it.